### ISSUE TITLE
``rating`` attribute

### ISSUE THREAD

**Issue by PyAres** [FujiMakoto/pygelbooru#14]
``rating`` (  that can be ``safe`` and ``explicit``)  should be a must for a good filtering, excluding tags are pretty much useless for safe searching, i had in mind implementing this but after several tests SFW image searching is an absolute no no, feels like the ignored tags have no effect since there will always be posts without nude tag but still be nsfw, ``pygelbooru``  it's a pretty simple and good lib  and sadly lacking a lot of stuff and pretty much abandoned, i have no hopes that someone will even answer to this issue like  in the #11  case

**Comment by FujiMakoto** [comment]
@PyAres I've added the score attribute now, but I unfortunately don't see a way to use this to automatically filter results on Gelbooru's end like you can with tags,
https://gelbooru.com/index.php?page=wiki&s=view&id=18780

So you'd need to use the attribute to filter the search results you get at the moment, unless maybe there is a way to do this that's not documented yet (or I'm just missing something)

**Comment by ghendersonnc** [comment]
If still relevant, and if I am understanding correctly: `rating:general` `rating:sensitive` `rating:questionable` `rating:explicit` can be be used as tags. General would be completely SFW results.

**Comment by PyAres** [comment]
> If still relevant, and if I am understanding correctly: `rating:general` `rating:sensitive` `rating:questionable` `rating:explicit` can be be used as tags. General would be completely SFW results.

indeed, it's a recent feature, though, as far as i understand atm there's no way of filtering the pics by rating alone ( i think? this is worth checking https://gelbooru.com/index.php?page=wiki&s=&s=view&id=25921 )

**Comment by ghendersonnc** [comment]
You can put `rating:general` into the tags and it will only give you SFW results
